flang/lib/Lower/OpenMP.cpp

```diff
@@ -120,7 +120,6 @@ class DataSharingProcessor {
   llvm::SetVector<const Fortran::semantics::Symbol *> privatizedSymbols;
   llvm::SetVector<const Fortran::semantics::Symbol *> defaultSymbols;
   llvm::SetVector<const Fortran::semantics::Symbol *> symbolsInNestedRegions;
-  llvm::SetVector<const Fortran::semantics::Symbol *> symbolsInParentRegions;
```

Comment:
Why is this not required anymore?

Reply:
Bad design from my end. Earlier, we were collecting both the symbol as well as the host-association details, and then removing one of them using this list. I have modified the current implementation to a use a `else if`, and hence this isn't required.